HelpSource/Classes/A2K.schelp

```diff
@@ -16,3 +19,25 @@ method::kr
 argument::in
 The input signal.
 
+Examples::
+
+code::
+(
+~signal   = { A2K.kr ( SinOsc.ar ( 1, 0, 1 ) ) };
+~signal2 = { A2K.kr ( LFDNoise0.ar ( 10, 1, 0 ) ) };
+~signal3 = { A2K.kr ( HenonC.ar ( 1 ) ) };
+~signal4 = { A2K.kr ( Dust.ar ( 1000 ) ) };
+)
+::
+
+code::
+~signal.scope;
+~signal2.scope;
+~signal3.scope;
+~signal4.scope;
+::
+
+code::
+// this may be deleted if you the community agrees. i understand people don't want to see these examples in the schelp, despite i think they are actually pretty useful, but that's a personal concern, simply
```

Comment:
I think we always want examples, but they need to be simple, not too simple, and bonus points if it is useful or inspiring.